source/_posts/2021-04-07-release-20214.markdown

```diff
@@ -951,7 +951,7 @@ Home Assistant Core 2021.7.0:
 
 - `speed_command_topic`
 - `speed_state_topic`
-- `state_value_template`
+- `speed_value_template`
 - `speed_list`
```

Comment:
The MQTT fan configuration option for `speed_list` attribute is `speeds`.
Further the following configuration options won't be needed anymore:
- `payload_off_speed`
- `payload_low_speed`
- `payload_medium_speed`
- `payload_high_speed`

The rest LGTM